metadata_repository_service/api/routers/analyses.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"Routes for retrieving Analyses"
+
+from fastapi import APIRouter, Depends
+from fastapi.exceptions import HTTPException
+
+from metadata_repository_service.config import Config, get_config
+from metadata_repository_service.dao.analysis import get_analysis
+from metadata_repository_service.models import Analysis
+
+analysis_router = APIRouter()
+
+
+@analysis_router.get(
+    "/analyses/{analysis_id}", response_model=Analysis, summary="Get an Analysis"
+)
+async def get_analyses(
+    analysis_id: str, embedded: bool = False, config: Config = Depends(get_config)
+):
+    """
+    Given an Analysis ID, get the Analysis record from the metadata store.
+    """
+    analysis = await get_analysis(analysis_id, embedded, config)
```

Comment:
Just one minor thing. Could we use kwargs here, that's in general a bit safer:
```suggestion
    analysis = await get_analysis(
        analysis_process_id=analysis_id,
        embedded=embedded,
        config=config,
    )
```

Reply:
Yes, good point 👍